src/components/githubDomainChecker.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+
+import {useState} from 'react';
+
+const MAX_COMPONENTS_ON_PAGE = 100;
+
+export function GitHubDomainChecker() {
+  const [domain, setDomain] = useState('');
+  const [isValidDomain, setIsValidDomain] = useState(false);
+
+  const isGitHubCom =
+    domain.toLowerCase().trim() === 'github.com' ||
+    domain.toLowerCase().trim() === 'https://github.com';
```

Comment:
I think maybe we should relax this a bit, if I add my GH user it recommends the wrong installation. probably user error by adding my username after but still!

<img width="953" alt="image" src="https://github.com/user-attachments/assets/02b00291-1dae-4f44-b46b-0a0c9174367a" />